swap_meet/vendor.py

```diff
@@ -1,2 +1,101 @@
+from swap_meet.item import Item
+
+# Wave 1
+"""The class vendor takes in a parameter of inventory. It has a method function
+    that adds and removes items.
+    Returns:
+      It returns the item or the boolean false.
+        """
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+        self.item = Item()
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+# Wave 2
+        """This method checks through the items csategory and appends it to a list.
+        """
+
+    def get_by_category(self, category):
+        items = []
+        for item in self.inventory:
+            if item.category == category:
+                items.append(item)
+        return items
```

Comment:
Great work! We can also use a list comprehension to make this list:

```suggestion
    def get_by_category(self, category):
        items = [item for item in self.inventory if item.category == category]
        return items
```

Reply:
Here's a resource to play around with list comprehension when you have the chance: https://www.w3schools.com/python/python_lists_comprehension.asp